Virtuoso-82-Special-Offers.ttl

```diff
@@ -6,985 +6,535 @@
 @prefix   license: <http://www.openlinksw.com/ontology/licenses#> .
 @prefix    oplsof: <http://www.openlinksw.com/ontology/software#> .
 @prefix      skos: <http://www.w3.org/2004/02/skos/core#> .
-@prefix    source: <https://www.openlinksw.com/data/turtle/virtuoso8/2020/Virtuoso-82-Special-Offers.ttl> .
-@prefix sourceDAV: <https://www.openlinksw.com/DAV/virtuoso2.openlinksw.com/data/turtle/virtuoso8/2020/Virtuoso-82-Special-Offers.ttl> .
+@prefix     oplic: <http://www.openlinksw.com/ontology/licenses#> .
+@prefix      wdrs: <http://www.w3.org/2007/05/powder-s#> . 
+@prefix       xhv: <http://www.w3.org/1999/xhtml/vocab#> . 
+@prefix        cc: <http://creativecommons.org/ns#> . 
+@prefix       owl: <http://www.w3.org/2002/07/owl#> . 
+@prefix    source: <https://www.openlinksw.com/data/turtle/virtuoso8/2024/Virtuoso-Offers-Licenses-Prices.ttl> .
+@prefix sourceDAV: <https://www.openlinksw.com/DAV/virtuoso2.openlinksw.com/data/turtle/virtuoso8/2024/Virtuoso-Offers-Licenses-Prices.ttl> .
```

Comment:
`source:` and `sourceDAV:` changes suggest file changes to `data/turtle/virtuoso8/2024/Virtuoso-Offers-Licenses-Prices.ttl` ... 
but this file remains `data/turtle/virtuoso8/2020/Virtuoso-82-Special-Offers.ttl`

Reply:
The values are:

@prefix    source: <https://www.openlinksw.com/data/turtle/virtuoso8/2020/Virtuoso-Offers-Licenses-Prices.ttl> .
@prefix sourceDAV: <https://www.openlinksw.com/DAV/virtuoso2.openlinksw.com/data/turtle/virtuoso8/2020/Virtuoso-Offers-Licenses-Prices.ttl> .